src/simulate.js

```diff
@@ -110,6 +111,40 @@ var parseStartPage = function (projectRoot) {
     return 'index.html'; // default uri
 };
 
+/**
+ *  Helper function to check if a file or directory exists
+ */
+var existsSync = function (filename) {
+    try {
+        /* fs.existsSync is deprecated
+           fs.statSync throws if the path does not exists */
+        fs.statSync(filename);
+        return true;
+    } catch (error) {
+        return false;
+    }
+};
+
+/**
+ *  Helper function to create a directory recursively
+ */
+var makeDirectoryRecursiveSync = function (dirPath) {
```

Comment:
I just noticed that there are some edge cases for this function: If you specify `Z:\` as the path, where `Z:\` does not exist, then this function will loop infinitely since `path.dirname("Z:\\") === "Z:\\"`